### ISSUE TITLE
Frunk and Trunk icons are backwards

### ISSUE THREAD

**Issue by YTWKLR** [Kemmey/Kemmey-TeslaWatch-Public#229]
I have a MYP and on the widget on my iPhone 11 Pro running newest iOS the Frunk icon accesses the Trunk and the Trunk icon accesses the Frunk.

**Comment by Kemmey** [comment]
You're absolutely right. It is only the label / confirmation text that is swapped - the action is correct. I'll have this fixed in a release as soon as apple can approve it. 

Thank you for reporting this, and I'm very sorry for this error!

**Comment by YTWKLR** [comment]
Hey Kim thanks for the fast response and sounds great and no rush as I appreciate the hard work!  I don’t have an Apple Watch but this is so worth it for the widget! 😊

I did notice that “seat heaters” doesn’t show up on the widget either even though I have it high enough on the list that it should be. Just an FYI.

Thank you again!
Torrey

**Comment by Kemmey** [comment]
Thanks! I'm honored!

The seat heaters and compass are not actions per se, as they do navigation in the app. That's why they are not rendered, as they are incompatible at this time. I will fix it at a later time, but did not have time for a good solution, as the user could have navigated to any point in the app at the time of launch.

**Comment by YTWKLR** [comment]
Absolutely and thanks for the reasoning and makes sense to me and living in AZ heat seaters aren’t used too often but we are in our brisk season at 40+ degree nights so I might actually have to use them. Hahaha.

Have a great weekend and thanks again for the hard work!

**Comment by Kemmey** [comment]
@YTWKLR out of curiosity, as this app is highly targeted towards watch users - where did you hear about it?

If you find it useful beyond the widget, do let me know if you have feature requests or improvements!

Cheers

**Comment by YTWKLR** [comment]
For sure in a Tesla FB group someone posted a screen shot of it and described it shortly so I checked it out in the App Store and decided to try it out. 😊

Sounds great and I’ll keep an eye out of things I might like to have especially compared to another app I have and let you know. 👍

**Comment by Kemmey** [comment]
1.1.1 is propagating in the store now

**Comment by YTWKLR** [comment]
Got it and it works!  🙌👏👏👏
